xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -192,7 +192,7 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
     frame->format = m_sampleFormat;
 
     result = avcodec_fill_audio_frame(frame, channels, m_sampleFormat,
-                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 0);
+                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 1);
```

Comment:
You compute the requested size as: channels * src_samples*av_get_sample_size.

src_samples: 2205
Channels: 2
Sample size: 8

This is put into avcodec_fill_audio_frame. Your alloced size Plays zero role here. Ffmpeg Computes this and fails, cause src_buffer needs to be at least some Bytes larger. So either the method is called with wrong src_buffer size or alignment needs to be 1 as 32 needs a mod 32 number as requested size number.